src/code/dialogs/checklist.js

```diff
@@ -94,9 +109,10 @@ const OperationChecklistDialog = WDialog.extend({
     this.sortable = this.getListDialogContent(
       operation,
       operation.links.concat(operation.markers),
-      this.sortable.sortBy,
-      this.sortable.sortAsc
```

Comment:
shouldn't `this.sortable.sortBy` match `localStorage` content here ?

Reply:
Yes, less changes the better. I believe it will work. Think this was a leftover from when I was fighting with the flip in the meaning of sortAsc == true.